test/unit/markdown-it/index.test.js

```diff
@@ -0,0 +1,61 @@
+const markdownIt = require('../../../src/lib/markbind/src/lib/markdown-it/');
+
+// test('renders 1', () => {
+//   const result = '<p>1</p>';
+//   const expected = '<p>1</p>';
+//   expect(result).toEqual(expected);
+// });
+
+// test('renders 2', () => {
+//   const result = markdownIt.render('# heading');
+//   const expected = '<h1>heading</h1>\n';
+//   expect(result).toEqual(expected);
+// });
+
+test('markdown-it renders a fenced code block', () => {
+  const source = [
```

Comment:
Should probably add a comment to explain why you had to split up the lines... does eslint complain if you just do

````
   const source = `\`\`\`xml
some lines
some lines
```;
````